docs-ref-conceptual/Latest-version/includes/cli-install-linux-dnf.md

```diff
@@ -22,7 +22,19 @@ To install the Azure CLI, follow these steps:
    sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
```

Comment:
For RHEL 10, the public key becomes `https://packages.microsoft.com/keys/microsoft-2025.asc`

Reply:
@mikefrobbins the installation guide is correct. After fixing this issue, the doc is ready to merge.